src/inverseRobot.js

```diff
@@ -7,7 +7,26 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const revers = {};
+  const keys = Object.keys(robot);
+  const values = Object.values(robot);
+
+  for (let i = 0; i < keys.length; i++) {
+    if (keys.includes(keys[i], i + 1)) {
+      return null;
+    }
+  }
+
+  for (let i = 0; i < values.length; i++) {
+    if (values.includes(values[i], i + 1)) {
+      return null;
+    }
+  }
 
+  for (const key in robot) {
+    revers[robot[key]] = key;
```

Comment:
You can also use in operator to check if object has some property, by doing it here you can remove two loops above
Also you may check it via reverse[robot[key]] === undefined or revers.hasOwnProperty(robot[key])
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty
```suggestion
    if(robot[key] in revers) {
      return null
    }

    revers[robot[key]] = key;
```

Reply:
Thank you very much